delete_student.py

```diff
@@ -0,0 +1,25 @@
+def random_st_delete(list_of_student):
+    for i in range(len(list_of_student)):
```

Comment:
`range(len(x))` is considered bad practice in Python. See this [link](https://python-forum.io/Thread-Basic-Never-use-for-i-in-range-len-sequence) for the description and alternatives. 

**TLDR**: use `enumerate`